src/shared/components/Gallery/useGallery.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react'
+
+export default function useCarousel(list) {
+   console.log(list)
+   const [images, setImages] = useState(list)
+   const [current, setCurrent] = useState(0)
+
+   useEffect(() => {
+      if (Array.isArray(list)) {
+         setImages(list)
+      } else {
+         return
+      }
```

Comment:
```suggestion
      if(Array.isArray(list)) {
         setImages(list)
      }
```

A return is uneccesary in useEffect since we're not expecting a value from it

Reply:
I have made the changes kindly re-review this.